src/OhmmsSoA/VectorSoaContainer.h

```diff
@@ -26,34 +26,19 @@
 namespace qmcplusplus
 {
 /** SoA adaptor class for ParticleAttrib<TinyVector<T,D> >
-   * @tparm T data type, float, double, complex<float>, complex<double>
-   * @tparm Alloc memory allocator
-   */
+ * @tparm T data type, float, double, complex<float>, complex<double>
+ * @tparm Alloc memory allocator
+ */
 template<typename T, unsigned D, size_t ALIGN = QMC_CLINE, typename Alloc = Mallocator<T, ALIGN>>
 struct VectorSoaContainer
 {
-  using Type_t    = TinyVector<T, D>;
-  using Element_t = T;
-  /////alias to ParticleAttrib<T1,D>
-  //template <class T1> using PosArray = ParticleAttrib<TinyVector<T1,D> >;
-  ///number of elements
-  size_t nLocal;
-  ///number of elements + padded
-  size_t nGhosts;
-  ///number of elements allocated by myAlloc
-  size_t nAllocated;
-  ///pointer: what type????
-  T* myData;
-  ///allocator
-  Alloc myAlloc;
+  using AoSElement_t = TinyVector<T, D>;
+  using Element_t    = T;
```

Comment:
Not in general a fan of the _t is it really needed? We should discuss, here I  don't care its better than before.

Reply:
I was just intended to replace the old name which can't self-explained.
I don't have preference and will just follow the final choice.